rpc_jobs/multi_node_aio.yml

```diff
@@ -34,6 +34,24 @@
       - 'OnMetal-Multi-Node-AIO_{series}-{context}-{trigger}'
       - 'OnMetal-Multi-Node-AIO-Merge-Trigger_{series}'
 
+- project:
+    name: 'Leapfrog-Multi-Node-AIO-Jobs'
+    series:
+      - kilo:
+          branch: kilo
+          OPENSTACK_ANSIBLE_BRANCH: "stable/kilo"
+          TEST_SET: ""
```

Comment:
The job currently isn't building because USER_VARS isn't set.  Also, we don't use TEST_SET anymore, so you'll want to change TEST_SET to something like:

```
USER_VARS: |
    tempest_test_sets: 'all'
```
We run `all` for newton, though I'm not sure what set will pass against kilo.
 
https://github.com/weezer/rpc-gating/blob/5cf7600c649a40e22a351e9c5e7b1060ff7acb03/rpc_jobs/multi_node_aio.yml#L10-L11